tests/tools.vitruv.domains.java.ui.tests/pom.xml

```
@@ -7,6 +7,7 @@
 		<groupId>tools.vitruv</groupId>
 		<artifactId>tests</artifactId>
 		<version>0.3.0-SNAPSHOT</version>
+		<relativePath>..\.polyglot.pom.tycho</relativePath>
```

Why did this become necessary?

---

This became necessary because I could remove the `pom.xml` in the tests folder after the cleanup (using the pomless Tycho build extension), but then you have to explicitly reference the build-temporary pom (`.polyglot.pom.tycho`) from custom `pom.xml` in contained project folders, as otherwise it looks for the default `pom.xml`.

---

```suggestion
		<relativePath>../.polyglot.pom.tycho</relativePath>
```


If the location and name are documented behaviour, I am fine with the change. 
However, maybe adjust the path to Unix style?